src/blueberrypy/shell.py

```diff
@@ -0,0 +1,70 @@
+import atexit
+import importlib
+
+from blueberrypy.config import BlueberryPyConfiguration
+
+
+def get_user_namespace(config, include_pkg=False):
+    assert isinstance(config, BlueberryPyConfiguration), type(config)
+
+    ns = {'__name__': 'blueberrypy-shell'}
+
+    pkg_name = get_package_name(config)
+
+    if include_pkg:
+        app_module = importlib.import_module(pkg_name)
+        if hasattr(app_module, '__all__'):
+            for name in app_module.__all__:
+                ns[name] = getattr(app_module, name)
+        else:
+            for name, obj in vars(app_module).viewitems():
+                if not name.startswith("_"):
+                    ns[name] = obj
+
+    ns['cherrypy'] = importlib.import_module("cherrypy")
+    ns['blueberrypy'] = importlib.import_module("blueberrypy")
+
+    if config.use_sqlalchemy:
```

Comment:
I think you could reuse this property https://github.com/open-craft-guild/blueberrypy/blob/master/src/blueberrypy/config.py#L270